### PR TITLE
Rollout 2018 conferences

### PR DESCRIPTION
**Pull request by hola-soy-milk**
This rolls out 2018's conferences, along with their images and footer.

### CODE REVIEW

File: students/conferences.md [hunk 1]
```diff
@@ -83,17 +83,17 @@ We are not able to support our students with travel and accommodation. If you ha
...
+A huge **thank you** to all conferences who supported our students and provided free tickets during Rails Girls Summer of Code 2018.
 
 <section class="conferences-container">
 
   {% for conference in site.data.conferences %}
     <div class="conference-item">
       <p class="conference-logo">
         <a href="{{conference.url}}">
-        <img src="{{conference.img}}"></a>
+        <img width="200" height="200" src="{{conference.img}}"></a>
```
Comment (alicetragedy) [inline students/conferences.md]: could we move this styling bit into the css? maybe into `layout.scss` for now until we refactor/clean up those files 😇

**Review by alicetragedy — CHANGES_REQUESTED** [review]
Hey @ramonh! This all looks great, thanks for your work getting this up and running and apologies again for the late review. I'd love to continue putting all css in the css files and keep all markdown files as “styling free” as possible. I've left a small comment about moving one line out of `conferences.md` into a css file, I'd be great to take care of that before we merge. Thank you! ✨

File: students/conferences.md (hunk 1 above)
Comment (hola-soy-milk) [reply students/conferences.md]: Thanks for the feedback! 

Hmm, I'm not sure, but I don't think this is styling but image properties... not sure we can extract these into CSS. 

I could be wrong. Will have a look

Comment (alicetragedy) [reply students/conferences.md]: Are all images resized ahead of time to have exactly those sizes? (sorry, the PR wasn't showing me the conference logos) — if that's the case, then yes, you're right and it's the image properties that need to be defined in the html tag. If we need to “resize” the image (i.e. if it's not exactly the size we want it to be) then that would fall under styling and need to be moved to the css. I thought we were dealing with the latter. Could you confirm that's not the case? If so, we can go ahead and merge this!

Comment (hola-soy-milk) [reply students/conferences.md]: AH! Yes, good point, thanks @alicetragedy. I should be doing the latter. On it!

**Comment by hola-soy-milk** [comment]
Finally done! @alicetragedy Could you please take another look?

**Review by alicetragedy — APPROVED** [review]
Hey @ramonh, thank you for the changes! I think it might be better to have that snippet of css code in `layout.scss` instead of `style.scss` (because of the way our scss files are structured). However, I also think that might be outside of the scope of this PR, so I'd say we can merge this, and move the css to the correct place later. Thank you for taking the time to do this, looking forward to having this great change made! ✨

**Pull request merged**